src/main/java/org/folio/inventory/dataimport/util/AdditionalFieldsUtil.java

```diff
@@ -273,7 +274,7 @@ public static void move001To035(Record srcRecord) {
   }
 
   public static void normalize035(Record srsRecord) {
-    List<Subfield> subfields = get035SubfieldValues(srsRecord, TAG_035, TAG_035_SUB);
+    List<Subfield> subfields = get035OclcSubfieldValues(srsRecord, TAG_035, TAG_035_SUB);
```

Comment:
In my opinion, the name get035SubfieldOclcValues would be clearer because 035O can be misread without IDE.